src/polyhedralGravity/input/ConfigSource.h

```diff
@@ -37,6 +37,12 @@ namespace polyhedralGravity {
          */
         virtual std::vector<std::array<double, 3>> getPointsOfInterest() = 0;
 
+        /**
+         * Returns the enablement status of the input polyhedron sanity check
```

Comment:
I don't think `enablement` is the word you are looking for. It's more in terms of when you enable somebody to do something. E.g. GitHub enables us to do code review.

getInputCheckStatus ? getInputCheckActivation or something like that?